expression/partition_pruner.go

```diff
@@ -165,7 +182,7 @@ func newHashPartitionPruner() *hashPartitionPruner {
 
 // solve eval the hash partition expression, the first return value represent the result of partition expression. The second
 // return value is whether eval success. The third return value represent whether the eval result of partition value is null.
```

Comment:
1. what does `conflict` mean?
2. The comment for the third value needs to be updated?

Reply:
conflict means condition like `a = 5 and a = 3` which will lead to filter condition to be false.

Reply:
maybe `isAlwaysFalse` is a better name